issuer-service/src/main/java/ch/admin/bj/swiyu/issuer/common/exception/DemonstratingProofOfPossessionError.java

```diff
@@ -0,0 +1,20 @@
+package ch.admin.bj.swiyu.issuer.common.exception;
+
+import lombok.Getter;
+
+public enum DemonstratingProofOfPossessionError {
```

Comment:
You could use:

```suggestion
@Getter
@RequiredArgsConstructor
public enum DemonstratingProofOfPossessionError {
```

And remove the constructor